Objective-C/Tests/LogTest.m

```diff
@@ -367,13 +308,10 @@ - (void) testFileLoggingReEnableLogging {
 }
 
 - (void) testFileLoggingHeader {
-    CBLLogFileConfiguration* config = [self logFileConfig];
-    config.usePlainText = YES;
-    CBLDatabase.log.file.config = config;
-    CBLDatabase.log.file.level = kCBLLogLevelVerbose;
+    CBLLogSinks.file = [[CBLFileLogSink alloc] initWithLevel: kCBLLogLevelVerbose directory: logFileDirectory usePlaintext: YES maxKeptFiles: kCBLDefaultFileLogSinkMaxKeptFiles maxFileSize: kCBLDefaultLogFileMaxSize];
```

Comment:
Break it down to multiple lines. Generally, please keep the line up to around 120 chars.